src/tests/test_CA/Makefile.am

```diff
@@ -119,6 +119,30 @@ softhsm2_one.conf:
 	@echo "objectstore.backend = file" >> $@
 	@echo "slots.removable = true" >> $@
 
+softhsm2_mech_rsa_pkcs: softhsm2_mech_rsa_pkcs.conf
```

Comment:
Hi,

thanks for the test. You do not have to create a new token which is the same as `softhsm2_one`, you can use the path to the `softhsm2_one` directory in the 2 new conf files and add them as addition per-requisites to the `softhsm2_one` target.

bye,
Sumit

Reply:
Indeed, thanks.

Reply:
Fixed.